### PR TITLE
Allow non-required fields to be None

### LINKED ISSUE #70: No robots listed when schema verification fails on non-required fields
### Issue
Running the latest commit (34b5941), I am unable to list any robots because the response verification fails.

### Steps to recreate
Running
```
account = pybotvac.Account(pybotvac.PasswordSession(email=<email>, password=<password>))
for robot in list(account.robots):
  ...
```
on my account prints the following
 ```
Bad response from robots endpoint: expected str for dictionary value @ data['timezone']. Got: {'serial': 'OPSblabla', ... 'timezone': None, ...}
```
and the for loop returns.

### Explanation
The reason for this appears to be that `self._robots.add(robot_object)` is being skipped if response verification throws in `refresh_robots()` in account.py, regardless of whether the exception is due to a required or non-required field missmatch:
```
    def refresh_robots(self):
        """
        Get information about robots connected to account.

        :return:
        """

        resp = self._session.get("users/me/robots")

        for robot in resp.json():
            _LOGGER.debug("Create Robot: %s", robot)
            try:
                ROBOT_SCHEMA(robot)
                robot_object = Robot(
                    name=robot["name"],
                    vendor=self._session.vendor,
                    serial=robot["serial"],
                    secret=robot["secret_key"],
                    traits=robot["traits"],
                    endpoint=robot["nucleo_url"],
                )
                self._robots.add(robot_object)
            except MultipleInvalid as ex:
                # Robot was not described accordingly by neato
                _LOGGER.warning(
                    "Bad response from robots endpoint: %s. Got: %s", ex, robot
                )
                continue
...
```

### Proposed fixes
1. `ROBOT_SCHEMA` should allow `None` as timezone entry
2. Verification of a non-required field should create a log entry, but the rest of the code should execute as normal. If this is not possible, non-required fields should not be verified.

### PR DESCRIPTION
**Pull request by Santobert**
Edit `ROBOT_SCHEMA`, `MAP_SCHEMA` and `PERSISTENT_MAP_SCHEMA` to allow non-required values to be None.

Closes: #70 

This is just a suggestion. We can be more or less restrictive. For example, we could allow None only for undocumented fields and for those that we know can be None.

### CODE REVIEW

File: pybotvac/account.py [hunk 1]
```diff
@@ -29,52 +29,52 @@
...
-        "version": int,
+        "version": Any(int, None),
         Required("id"): str,
         Required("url"): Url,
-        "url_valid_for_seconds": int,
-        Optional("run_id"): str,  # documented, but  not present
+        "url_valid_for_seconds": Any(int, None),
+        Optional("run_id"): str,  # documented, but not present
         "status": Any("complete", "incomplete", "canceled"),
         "launched_from": Any("robot", "schedule", "app"),
```
Comment (Santobert) [inline pybotvac/account.py]: Should we edit the fields `status` and `launched_from` to be less restrictive as well?

```suggestion
        "status": Any(str, None),
        "launched_from": Any(str, None),
```

**Review by Santobert — COMMENTED** [review]
One additional question

File: pybotvac/account.py (hunk 1 above)
Comment (stianaske) [reply pybotvac/account.py]: I'd say that unless we depend on the value of the fields in downstream code, we should be as permissive as possible. As far as I can tell that is the case for both `status` and `launched_from`.

Comment (Santobert) [reply pybotvac/account.py]: I agree. Nice!

**Pull request merged**